qemu_aarch64_test.go

```diff
@@ -0,0 +1,67 @@
+package qcli
+
+import (
+	"strings"
+	"testing"
+)
+
+var (
+	fullUefiAarchVM = "-machine virt,accel=kvm -cpu host -m 1G -drive file=udisk.img,id=hd0,if=none,format=qcow2 -device virtio-blk-pci,drive=hd0,serial=hd0,disable-modern=false,addr=0x1e,bus=pcie.0,scsi=off,config-wce=off -drive file=ubuntu-22.04.2-live-server-arm64.iso,id=cdrom0,if=none,format=raw,media=cdrom,readonly=on -device virtio-blk-pci,drive=cdrom0,serial=cdrom0,bootindex=0,disable-modern=false,addr=0x1d,bus=pcie.0,scsi=off,config-wce=off -drive if=pflash,format=raw,readonly=on,file=/usr/share/AAVMF/AAVMF_CODE.ms.fd -drive if=pflash,format=raw,file=flash-vars-sec.img -object memory-backend-ram,id=dimm1,size=1G -numa node,memdev=dimm1 -nographic"
+)
+
+func fullVMConfigArch() *Config {
+	c := &Config{
+		Machine: Machine{
+			Type:         MachineTypeVirt,
+			Acceleration: MachineAccelerationKVM,
+		},
+		CPUModel:      "host", //cortex-a53, cortex-a57, cortex-a72
```

Comment:
Is the comment correct?  I believe you said that if you use KVM, then CPUModel must be "host"?

If so, we should see where in qemu.go we append the CPU model and put a check there.  And drop the comment here and in qemu_aarch64.go, 

I wonder if we should also introduce  a cpu_<arch>.go and populate those with values from `qemu-system-$arch -cpu ?`